src/ui/OGMessageItemBody/index.tsx

```diff
@@ -37,8 +37,15 @@ export default function OGMessageItemBody({
 }: Props): ReactElement {
   const imageRef = useRef<HTMLDivElement>(null);
   const { stringSet } = useContext(LocalizationContext);
+
   const openOGUrl = (): void => {
-    if (message?.ogMetaData?.url) window.open(message?.ogMetaData?.url);
+    let url = message?.ogMetaData?.url;
+    if (url) {
+      if (!url.startsWith('http')) {
+        url = 'https://' + url;
+      }
+      window.open(url);
```

Comment:
@bang9 I wanted to add try catch logic here but since this component is under `ui`, it has not senbrid provider context so we cannot import logger here. Any other suggestion you might have?

Reply:
It's likely that the `window.open` won't throw an error.